stock_release_channel_shipment_advice_deliver/models/stock_release_channel.py

```diff
@@ -63,6 +64,7 @@ def _compute_is_action_deliver_allowed(self):
                 )
                 and bool(rec.picking_to_plan_ids)
                 and rec.shipment_planning_method != "none"
+                and rec.auto_deliver
```

Comment:
hi @jbaudoux `auto_deliver` is added in `_compute_is_action_deliver_allowed`, please review it